DQM/SiPixelPhase1RawData/python/SiPixelPhase1RawData_cfi.py

```diff
@@ -69,7 +69,7 @@
   name = "nerrors_per_type",
   title = "Number of Errors per Type",
   xlabel = "Error Type",
-  range_min = -0.5, range_max = 50.5, range_nbins = 51,#TODO: proper range here
+  range_min = 24.5, range_max = 40.5, range_nbins = 16,#TODO: proper range here
```

Comment:
i'm naively surprised that this would not start at 0. There are no errors of # 0 to 24? Otherwise, can the comment now be removed? @enochnotsocool